aspnetcore/blazor/hybrid/class-libraries.md

```diff
@@ -0,0 +1,514 @@
+
+  ```diff
+  + @using {RCL PACKAGE ID/ASSEMBLY}
+  + @using {RCL PACKAGE ID/ASSEMBLY}.Shared
+  ```
+
+* If the Blazor Server app's `Shared` folder is empty because all of the assets are moved to the RCL, *update* the `Shared` folder namespace to the RCL's `Shared` folder namespace, along with adding the RCL's namespace:
+
+  ```diff
+  - @using {BLAZOR SERVER ASSEMBLY}.Shared
+  + @using {RCL PACKAGE ID/ASSEMBLY}
+  + @using {RCL PACKAGE ID/ASSEMBLY}.Shared
+  ```
+
+For example:
+
+```razor
+...
+@using SharedLibrary
+@using SharedLibrary.Shared
+```
+
+Update or add the namespace of the RCL in the Blazor Hybrid's `_Imports.razor` file:
+
+* If the Blazor Hybrid app's `Shared` folder remains in the app because the project exclusively uses remaining components in the folder, then *add* the RCL's `Shared` folder namespace to the file, along with adding the RCL's namespace:
+
+  ```diff
+  + @using {RCL PACKAGE ID/ASSEMBLY}
+  + @using {RCL PACKAGE ID/ASSEMBLY}.Shared
+  ```
+
+* If the Blazor Hybrid app's `Shared` folder is empty because all of the assets are moved to the RCL, *update* the `Shared` folder namespace to the RCL's `Shared` folder namespace, along with adding the RCL's namespace:
+
+  ```diff
+  - @using {BLAZOR HYBRID ASSEMBLY}.Shared
+  + @using {RCL PACKAGE ID/ASSEMBLY}
+  + @using {RCL PACKAGE ID/ASSEMBLY}.Shared
+  ```
+
+For example:
+
+```razor
+...
+@using SharedLibrary
+@using SharedLibrary.Shared
+```
+
+Place additional shared code and static assets in the RCL:
+
+* Images, JavaScript (`.js`), and stylesheets (`.css`): Place shared static assets into the RCL's `wwwroot` folder.
+
+  The app's stylesheet (`wwwroot/css/app.css` or `wwwroot/css/site.css`) can be moved from the Blazor Hybrid project or the Blazor Server project to the RCL, as can the Bootstrap assets (`wwwroot/css/bootstrap` folder) and open iconic assets (`wwwroot/css/open-iconic` folder). After placing the shared assets into the RCL, they can be removed from the Blazor Hybrid and Blazor Server projects if they aren't referenced further by the apps.
+
+  The Blazor Server app's `Pages/_Layout.cshtml` file and the Blazor Hybrid app's `wwwroot/index.html` file are updated to access these assets from the RCL. Remove the existing entries in the files for the local Bootstrap and stylesheet assets, as they're no longer in the individual apps after they're moved to the RCL.
+
+  ```html
+  <link rel="stylesheet" href="_content/{PACKAGE ID/ASSEMBLY}/css/bootstrap/bootstrap.min.css" />
+  <link href="_content/{PACKAGE ID/ASSEMBLY}/css/app.css" rel="stylesheet" />
+  ```
+
+  For an RCL named `SharedLibrary`:
+
+  ```html
+  <link rel="stylesheet" href="_content/SharedLibrary/css/bootstrap/bootstrap.min.css" />
+  <link href="_content/SharedLibrary/css/app.css" rel="stylesheet" />
+  ```
+
+  > [!NOTE]
+  > In a Blazor WebAssembly app, the preceding `<link>` tags are in the `wwwroot/index.html` file.
+
+* Place additional shared components and C# code into the RCL.
```

Comment:
While the weather forecast service in the Blazor project templates is the same for Blazor Server and Blazor Hybrid scenarios, it's different for Blazor WebAssembly, which makes an HTTP request for the data. Since this content is very project template specific and you already have a section below on dealing with services that are hosting model specific, I think this content can be removed.